helm/cert-manager/charts/cert-manager-giantswarm-netpol/templates/waitforcilium-job.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: waitforcilium
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "labels" . | nindent 4 }}
+  annotations:
+    helm.sh/hook: pre-install,pre-upgrade
+    helm.sh/hook-weight: "-1"
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+spec:
+  backoffLimit: 50
+  template:
+    metadata:
+      labels:
+        {{- include "labels" . | nindent 8 }}
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: waitforcilium
+      priorityClassName: system-cluster-critical
+      containers:
+        - name: wait-for-cilium-crds
+          image: "quay.io/giantswarm/docker-kubectl:1.24.4"
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              cpu: 100m
+              memory: 50Mi
+            limits:
+              cpu: 200m
+              memory: 100Mi
+          securityContext:
+            readOnlyRootFilesystem: true
+            runAsUser: 1000
+            runAsNonRoot: true
+          command:
+            - sh
+          args:
+            - -c
+            - |
+              kubectl wait --timeout=5m --for=condition=established crd ciliumnetworkpolicies.cilium.io
+              kubectl wait --timeout=1m --for=condition=established crd ciliumclusterwidenetworkpolicies.cilium.io
```

Comment:
```zsh
% kubectl wait crd ciliumnetworkpolicies.cilium.io ciliumclusterwidenetworkpolicies.cilium.io --for condition=established --timeout 5m
customresourcedefinition.apiextensions.k8s.io/ciliumnetworkpolicies.cilium.io condition met
customresourcedefinition.apiextensions.k8s.io/ciliumclusterwidenetworkpolicies.cilium.io condition met
```

Reply:
Yea but it still won't get to that point and fail before the job is executed, since it's blocked because of the missing CRDs. Once the CRDs are there the release will start to roll out but then you don't need the job anymore as it's job is to wait for the CRDs.